pghoard/inotify.py

```diff
@@ -96,21 +97,44 @@ def create_event(self, wd, mask, cookie, name):
         if mask & event_types['IN_IGNORED']:
             # explicit removal of watch or dir, ignore
             return
-        full_path = os.path.join(self.watch_to_path[wd], name.decode("utf8"))
-        if mask & event_types["IN_CREATE"] > 0:
-            self.compression_queue.put({"type": "CREATE", "full_path": full_path})
-        elif not self.ignore_modified and mask & event_types["IN_MODIFY"] > 0:  # no need for these
+
+        decoded_name = name.decode("utf8")
+        full_path = os.path.join(self.watch_to_path[wd], decoded_name)
+
+        def ev(ev_type):
+            if self.log.getEffectiveLevel() <= logging.DEBUG:
+                try:
+                    st = os.stat(full_path)
+                except:  # pylint: disable=bare-except
+                    st = None
+
+            self.log.debug("event: %s %s, %r", full_path, ev_type, st)
+
+        if not self.ignore_modified and mask & event_types["IN_MODIFY"] > 0:
```

Comment:
With this all the modify events go through all the elifs even if they're ignored.